.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,76 @@
+name: CI/CD
+
+on:
+  push:
+    branches: 
+    - master
+  pull_request:
+    branches: 
+    - master
+
+jobs:
+  test:
+    name: Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version:
+        - 10
+        - 12
+        os: 
+        - ubuntu-latest
+        - windows-latest
+        - macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setting up Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+
+    - name: Get npm cache directory
+      id: npm-cache
+      run: |
+        echo "::set-output name=dir::$(npm config get cache)"
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.npm-cache.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+          
+    - name: Installing npm modules
+      run: npm install
+
+    - name: Running Tests
+      run: npm run test
+
+    - name: Electron Builder Action
+      uses: samuelmeuli/action-electron-builder@v1.3.0
+      with:
+        github_token: ${{ secrets.github_token }}
+        mac_certs: ${{ secrets.mac_certs }}
+        mac_certs_password: ${{ secrets.mac_certs_password }}
+
+    - name: See files in dist folder
+      run: ls ./dist
+
+    - name: Cleanup artifacts
+      if: matrix.os != 'windows-latest'
+      run: |
+          mkdir artifacts
+          mv "dist/(*.exe,*.deb,*.AppImage,*.dmg)" artifacts || true
+
+    - name: Cleanup artifacts Win
+      if: matrix.os == 'windows-latest'
+      run: |
+          mkdir artifacts
+          mv dist/*.exe artifacts
+
+    - name: Deploying to GPR
```

Comment:
Name this step, `Uploading artifacts` or similar. "Deploying" and "GPR" and unrelated words to describe the current step.

Reply:
Yes, I changed the name. I am new to the action feature of GitHub, and I thought it will upload the artifact to GitHub Package Repository (GRP) but it has a different way to do this. Currently, we can use this artifact from the checks view of the pull request.